jmx-metrics/src/test/groovy/io/opentelemetry/contrib/jmxmetrics/target-systems/CassandraIntegrationTests.groovy

```diff
@@ -31,170 +30,164 @@ class CassandraIntegrationTests extends OtlpIntegrationTest  {
         Testcontainers.exposeHostPorts(otlpPort)
         configureContainers('target-systems/cassandra.properties',  otlpPort, 0, false)
 
-        expect:
-        when: 'we receive metrics from the JMX metrics gatherer'
-        List<ResourceMetrics> receivedMetrics = collector.receivedMetrics
-        then: 'they are of the expected size'
-        receivedMetrics.size() == 1
+        ArrayList<Metric> metrics
+        await().atMost(30, TimeUnit.SECONDS).untilAsserted {
```

Comment:
It seems like this made the tests fairly reliable. If some other flakes come up we can probably tweak those tests too